chart/kube-arangodb/templates/deployment.yaml

```diff
@@ -46,6 +46,19 @@ spec:
                           operator: In
                           values:
                             - amd64
+              podAntiAffinity:
+                requiredDuringSchedulingIgnoredDuringExecution:
```

Comment:
Please - change to preferredDuringSchedulingIgnoredDuringExecution

Reply:
changed